LoopOnboardingKitUI/View Controllers/OnboardingUICoordinator.swift

```diff
@@ -0,0 +1,425 @@
+//
+//  OnboardingUIController.swift
```

Comment:
@darinkrauss I'm also assuming this code is mostly copy/pasted here. If there is code you want to call out please note it. Thanks.